stress/shell/src/main/java/alluxio/stress/cli/Benchmark.java

```diff
@@ -167,13 +167,14 @@ public String run(String[] args) throws Exception {
    *
    * @param startMs the start time
    * @param endMs the end time
+   * @param isTTFB check if method is time to first byte function
    * @param nameTransformer function which transforms the type and method into a name. If the
    *                        function returns null, then the method is skipped
    * @return a map of names to statistics
    */
   @SuppressFBWarnings(value = "DMI_HARDCODED_ABSOLUTE_FILENAME")
   protected Map<String, MethodStatistics> processMethodProfiles(long startMs, long endMs,
-      BiFunction<String, String, String> nameTransformer) throws IOException {
+      boolean isTTFB, BiFunction<String, String, String> nameTransformer) throws IOException {
```

Comment:
Instead of passing in this via the parameters, can we just introduce a simple java class that just stores the json fields? Then, we can let the `nameTransformer` be a `Function<ProfileClass, String>` and the function implementation can deal with it how it chooses.

Reply:
changed to `Function<ProfileClass, String>`.